ClickHouse.Test/AdBroTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Diagnostics;
+using System.Linq;
+using ClickHouse.Ado;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace ClickHouse.Test
+{
+    [TestClass]
+    public class AdBroTests
```

Comment:
Let's just extend their test file or somehow omit `ADBRO` naming if we are planning to push it to the original project.